zerver/tests/tests.py

```diff
@@ -1182,7 +1182,7 @@ def test_patch_bot_avatar(self):
 
         # Try error case first (too many files):
         with open(os.path.join(TEST_AVATAR_DIR, 'img.png'), 'rb') as fp1, \
-             open(os.path.join(TEST_AVATAR_DIR, 'img.gif'), 'rb') as fp2:
+             open(os.path.join(TEST_AVATAR_DIR, 'img.gif'), 'rb') as fp2:  # NOQA: PEP8 E127 fails here
```

Comment:
It's worth investigating whether there's a better way we can write this code to not have this E127 error...